graph-manager-docs/source/schema-registry.mdx

```diff
@@ -5,23 +5,71 @@ description: A central hub for your data graph
 
 import ProjectConfigPanel from "gatsby-theme-apollo-docs/shared/project-config-panel.mdx";
 import VideoFrame from "../components/video-frame";
+import {
+  ObtainGraphApiKey,
+} from '../shared/obtain-graph-api-key';
 
-Apollo Graph Manager provides a **schema registry** that serves as a central hub for managing your data graph (as recommended by [Principled GraphQL](https://principledgraphql.com/integrity#3-track-the-schema-in-a-registry)). The schema registry enables you to explore your schema's types and fields, track its change history, and improve your production server's security by removing the need for introspection. You can also [register **variants** of your schema](#managing-environments-with-variants) to support environments besides production, such as test or staging.
+Apollo Graph Manager's **schema registry** is a central hub for managing your data graph (as recommended by [Principled GraphQL](https://principledgraphql.com/integrity#3-track-the-schema-in-a-registry)). The schema registry enables you to explore your schema's types and fields, track its change history, and improve your production server's security by removing the need for introspection. You can also [register **variants** of your schema](#managing-environments-with-variants) that correspond to different environments, such as staging and production.
 
-<VideoFrame
-  title="Tracking your GraphQL Schema in a Registry"
-  src="https://www.youtube.com/embed/duwM95RsiBs"
-/>
+## Automatic registration
 
-## Prerequisites
+> **Automatic schema registration is not available for graphs that use Apollo Federation.** If you have a federated graph, instead see [Setting up managed federation](./managed-federation/setup/).
 
-To get started with the schema registry, first complete the following setup for the project that includes your GraphQL server:
+### Apollo Server setup
 
-<ProjectConfigPanel />
+Apollo Server version 2.15.1 and later can register its schema with Graph Manager automatically every time it starts up. To do so:
+
+1. Obtain an API key for your data graph:
+
+    <ObtainGraphApiKey />
+
+2. Set this API key as the value of the `APOLLO_KEY` environment variable in your server's environment.
+
+    If you're using the [`dotenv`](https://www.npmjs.com/package/dotenv) library, you can add the API key to the `.env` file in your project's root directory, like so:
+
+    ```none:title=.env
+    APOLLO_KEY=YOUR_KEY_HERE
+    ```
+
+3. Set the `reportSchema` option to `true` in your `ApolloServer` constructor, like so:
+
+    ```js
+    const server = new ApolloServer({
+      // ...other options...
+      reportSchema: true,
+    });
+    ```
+
+Now every time your server starts up, it registers its schema with Graph Manager.
```

Comment:
Should we explain the nuance here that we wait for deploys to finish or is that left to the more intrepid reader?